packages/waku/src/environment/index.worker.ts

```diff
@@ -0,0 +1,6 @@
+export function getEnv(key: string): string | undefined {
+  if (!(globalThis as any).__WAKU_PRIVATE_ENV__) {
+    return undefined;
+  }
```

Comment:
This shouldn't return `undefined` as expected, but waku is using `workerd` instead of `worker` now

https://github.com/dai-shi/waku/pull/394